UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling.cs

```diff
@@ -16,6 +17,28 @@ public class Changeling : Antagonist
 	{
 		public override Mind ServerSpawn(PlayerSpawnRequest spawnRequest)
 		{
+			if (spawnRequest.CharacterSettings.Species.ToLower().Contains("cow") || spawnRequest.CharacterSettings.Species.ToLower().Contains("monkey"))
```

Comment:
would be better to add a tag on the speciesSo saying it's some special type, also why is monkey not allowed?

Reply:
why is monkey not allowed - 
Because monkey have their own special brain. 
would be better to add a tag on the speciesSo - 
Yep, gonna change that